crates/bevy_ecs/src/system/exclusive_function_system.rs

```diff
@@ -10,7 +10,7 @@ use crate::{
     world::{unsafe_world_cell::UnsafeWorldCell, World},
 };
 
-use alloc::borrow::Cow;
+use alloc::{borrow::Cow, vec, vec::Vec};
```

Comment:
I think this would usually be written as `vec::{self, Vec}`.

Reply:
Unfortunately this doesn't work. `vec::{self}` imports the _module_ `vec`, while `vec` imports the _macro_ `vec!`.

Reply:
ah if it's the macro then it is okay!